backend/pom.xml

```diff
@@ -61,7 +99,16 @@
 				<artifactId>versions-maven-plugin</artifactId>
 				<version>2.17.1</version>
 			</plugin>
+			<plugin>
```

Comment:
Eu não to vendo problema em você mergear isso, pq eu comparei os dois arquivos e estão iguais, Mas na resolução de conflito onde vc escolhe suas modificações, ao invés das modificações que da main, corretamente, corre o risco de você sobrescrever coisas. Então é preciso saber sempre, o que é seu, e o que é da main, e como isso deve conviver.